src/Microsoft.ML.Data/Transforms/GenerateNumberTransformer.cs

```diff
@@ -13,11 +13,11 @@
 using System.Collections.Generic;
 using Float = System.Single;
 
-[assembly: LoadableClass(GenerateNumberTransform.Summary, typeof(GenerateNumberTransform), typeof(GenerateNumberTransform.Arguments), typeof(SignatureDataTransform),
-    GenerateNumberTransform.UserName, GenerateNumberTransform.LoadName, "GenerateNumber", GenerateNumberTransform.ShortName)]
+[assembly: LoadableClass(GenerateNumberTransformer.Summary, typeof(GenerateNumberTransformer), typeof(GenerateNumberTransformer.Arguments), typeof(SignatureDataTransform),
```

Comment:
>GenerateNumberTransformer [](start = 25, length = 25)

Does this match ActionPerformingTransformer?

Reply:
Maybe NumberGeneraingTransformer instead?

---
In reply to: [233196694](https://github.com/dotnet/machinelearning/pull/1607#discussion_r233196694) [](ancestors = 233196694)

Reply:
NumbersGeneratingTransformer?

---
In reply to: [233216627](https://github.com/dotnet/machinelearning/pull/1607#discussion_r233216627) [](ancestors = 233216627,233196694)